openwisp_controller/config/admin.py

```diff
@@ -698,9 +710,55 @@ class Media(BaseConfigAdmin):
         js = list(BaseConfigAdmin.Media.js) + [f'{prefix}js/vpn.js']
 
 
+class DeviceGroupForm(BaseForm):
+    class Meta(BaseForm.Meta):
+        model = DeviceGroup
+        widgets = {'context': DeviceGroupJsonSchemaWidget}
+        labels = {'context': _('Metadata')}
+        help_texts = {
+            'context': _(
+                # TODO: Rephrase this
+                'In this section you can add metadata of the DeviceGroup'
+            )
+        }
+
+
+class DeviceGroupAdmin(BaseAdmin):
+    form = DeviceGroupForm
+    fields = [
+        'name',
+        'organization',
+        'description',
+        'context',
+        'created',
+        'modified',
+    ]
+    search_fields = ['name']
```

Comment:
you know what would be good here? A way to filter groups of a specific org, if you look in `DeviceAdmin` you'll see how to do this in a way that is multitenant-ready.

Reply:
Added automated tests as well. :)